test/tests/extras.test.ts

```diff
@@ -34,14 +34,20 @@ describe(`Extra Tests`, () => {
     });
 
     describe(`ts-node register script`, () => {
+      /** Yarn sometimes outputs bold text, which makes these tests flakey */
+      function stripAnsi(str: string) {
+        // eslint-disable-next-line no-control-regex
+        return str.replace(/\x1B\[[0-?]*[ -/]*[@-~]/g, "");
+      }
+
       test(`Works with --transpileOnly`, () => {
         const res = execSync("yarn g:ts-node --transpileOnly src/index.ts", { cwd: projectRoot }).toString();
-        expect(res).toMatch(/^null($|\r?\n)/m);
+        expect(stripAnsi(res.trim())).toEqual("null");
```

Comment:
Also, instead of caring about the newline (which can be different between windows and mac), I just `.trim()` it.